docs/troubleshooting/development/vtex-release-cli-command-returns-ebadf-message.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "`vtex release` CLI command returns an EBADF message"
+slug: "vtex-release-cli-command-returns-ebadf-message"
+hidden: false
+createdAt: "2024-08-09T13:00:00.000Z"
+updatedAt: "2024-08-09T13:00:00.000Z"
+excerpt: "Creating an app release using the VTEX IO CLI returns an EBADF error."
+tags:
+  - vtex-io
+  - cli
+  - release
+---
+
+**Keywords:** Release | CLI
+
+When running the [VTEX IO CLI command](https://developers.vtex.com/docs/guides/vtex-io-documentation-vtex-io-cli-command-reference#release) [vtex release](https://developers.vtex.com/docs/guides/vtex-io-documentation-vtex-io-cli-command-reference#release), it returns an EBADF (error bad file descriptor) message. This happens because a file descriptor variable does not correctly point to a file during command execution.
```

Comment:
```suggestion
When running the [VTEX IO CLI command](https://developers.vtex.com/docs/guides/vtex-io-documentation-vtex-io-cli-command-reference#release) [`vtex release`](https://developers.vtex.com/docs/guides/vtex-io-documentation-vtex-io-cli-command-reference#release), it returns an EBADF (error bad file descriptor) message. This happens because a file descriptor variable does not correctly point to a file during command execution.
```

Reply:
Bem notado! Isso é de uma falha do conversor markdown. Quando tem código dentro do link, ele separa em dois links e tira a formatação de `code`. Vou ver depois se corrijo isso.